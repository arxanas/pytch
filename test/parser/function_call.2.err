In test/parser/function_call.2.pytch, line 2, character 1:
EXPECTED_RPAREN[1003]: I was expecting a ')' to indicate the end of this function argument list, but instead got the end of the file.
   +-----------------------------------+
   | test/parser/function_call.2.pytch |
 1 | print(                            |
   +-----------------------------------+
