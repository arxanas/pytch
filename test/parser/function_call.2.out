SyntaxTree
    FunctionCallExpr
        IdentifierExpr
            Token 'print'
        ArgumentList
            Token '('
            Trailing '\n'
            <missing>
    Token ''
